### ISSUE TITLE
certbot renew is not working on OpenSUSE Leap

### ISSUE THREAD

**Issue by kajla** [certbot/certbot#4478]
## My operating system is (include version):
OpenSUSE Leap 42.2

## I installed Certbot with (certbot-auto, OS package manager, pip, etc):
OS package manager (https://build.opensuse.org/package/show?project=devel%3Alanguages%3Apython&package=certbot)

## I ran this command and it produced this output:
certbot renew

## Certbot's behavior differed from what I expected because:
It wrote this:
Saving debug log to /var/log/certbot/letsencrypt.log
Currently, the renew verb is capable of either renewing all installed certificates that are due to be renewed or renewing a single certificate specified by its name. If you would like to renew specific certificates by their domains, use the certonly command. The renew verb may provide other options for selecting certificates to renew in the future.

###### Logs are stored in `/var/log/letsencrypt` by default. Feel free to redact domains, e-mail and IP addresses as you see fit.
2017-04-07 21:21:30,271:DEBUG:certbot.main:Root logging level set at 20
2017-04-07 21:21:30,273:INFO:certbot.main:Saving debug log to /var/log/certbot/letsencrypt.log
2017-04-07 21:21:30,275:DEBUG:certbot.main:certbot version: 0.12.0
2017-04-07 21:21:30,275:DEBUG:certbot.main:Arguments: []
2017-04-07 21:21:30,276:DEBUG:certbot.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#nginx,PluginEntryPoint#standalone,PluginEntryPoint#manual,PluginEntryPoint#webroot,PluginEntryPoint#apache,PluginEntryPoint#null)
2017-04-07 21:21:30,278:DEBUG:certbot.main:Exiting abnormally:
Traceback (most recent call last):
  File "/usr/bin/certbot", line 11, in <module>
    load_entry_point('certbot==0.12.0', 'console_scripts', 'certbot')()
  File "/usr/lib/python2.7/site-packages/certbot/main.py", line 896, in main
    return config.func(config, plugins)
  File "/usr/lib/python2.7/site-packages/certbot/main.py", line 702, in renew
    renewal.handle_renewal_request(config)
  File "/usr/lib/python2.7/site-packages/certbot/renewal.py", line 370, in handle_renewal_request
    raise errors.Error("Currently, the renew verb is capable of either "
Error: Currently, the renew verb is capable of either renewing all installed certificates that are due to be renewed or renewing a single certificate specified by its name. If you would like to renew specific certificates by their domains, use the certonly command. The renew verb may provide other options for selecting certificates to renew in the future.

What can I do? :(
Thanks in advance.

**Comment by pde** [comment]
Did you possibly put your list of domain names in `cli.ini`? Though the docs previously suggested doing that, it's not actually compatible with the `renew` subcommand at present :/

**Comment by pde** [comment]
Your two options are:

1. Remove domains from `cli.ini`, and provide them on the command line when you make new certs.
2. Perform renewal with `certbot certonly -q` (or `-n` if you want more verbosity) instead of `certbot renew`.

**Comment by pde** [comment]
If so, this would be an instance of #4096.

**Comment by kajla** [comment]
Hi pde,

Yes, I added my domain name to `cli.ini`.
After deleted, certbot renew is working:

```
# certbot renew
Saving debug log to /var/log/certbot/letsencrypt.log

-------------------------------------------------------------------------------
Processing /etc/certbot/renewal/mydomain.conf
-------------------------------------------------------------------------------
Cert not yet due for renewal

The following certs are not due for renewal yet:
  /etc/certbot/live/mydomain/fullchain.pem (skipped)
No renewals were attempted.
```

/var/log/certbot/letsencrypt.log log file:
```
2017-04-09 10:36:01,573:DEBUG:certbot.main:Root logging level set at 20
2017-04-09 10:36:01,575:INFO:certbot.main:Saving debug log to /var/log/certbot/letsencrypt.log
2017-04-09 10:36:01,576:DEBUG:certbot.main:certbot version: 0.12.0
2017-04-09 10:36:01,576:DEBUG:certbot.main:Arguments: []
2017-04-09 10:36:01,578:DEBUG:certbot.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#nginx,PluginEntryPoint#standalone,PluginEntryPoint#manual,PluginEntryPoint#webroot,PluginEntryPoint#apache,PluginEntryPoint#null)
2017-04-09 10:36:01,610:INFO:certbot.renewal:Cert not yet due for renewal
2017-04-09 10:36:01,611:DEBUG:certbot.renewal:no renewal failures
```

And yes, it is also working with --nginx:
```
# certbot renew --nginx
Saving debug log to /var/log/certbot/letsencrypt.log

-------------------------------------------------------------------------------
Processing /etc/certbot/renewal/mydomain.conf
-------------------------------------------------------------------------------
Cert not yet due for renewal

The following certs are not due for renewal yet:
  /etc/certbot/live/mydomain/fullchain.pem (skipped)
No renewals were attempted.
```

/var/log/certbot/letsencrypt.log log file:
```
2017-04-09 10:37:50,142:DEBUG:certbot.main:Root logging level set at 20
2017-04-09 10:37:50,152:INFO:certbot.main:Saving debug log to /var/log/certbot/letsencrypt.log
2017-04-09 10:37:50,153:DEBUG:certbot.main:certbot version: 0.12.0
2017-04-09 10:37:50,153:DEBUG:certbot.main:Arguments: ['--nginx']
2017-04-09 10:37:50,155:DEBUG:certbot.main:Discovered plugins: PluginsRegistry(PluginEntryPoint#nginx,PluginEntryPoint#standalone,PluginEntryPoint#manual,PluginEntryPoint#webroot,PluginEntryPoint#apache,PluginEntryPoint#null)
2017-04-09 10:37:50,208:INFO:certbot.renewal:Cert not yet due for renewal
2017-04-09 10:37:50,208:DEBUG:certbot.renewal:no renewal failures
```

Thank you for your help! 😄

**Comment by pde** [comment]
Ok, closing this as a duplicate of #4096 (which we've also fixed for the forthcoming 0.14.0 release)
